### ISSUE TITLE
TwoSceneSelectionNode should be a RadioButtonGroup

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/balloons-and-static-electricity#470]
`TwoSceneSelectionNode` should be a `RadioButtonGroup`. I'm surprised to find that it's not, given the a11y and phet-io work that's been done on this sim. And now it's impeding work on https://github.com/phetsims/scenery/issues/1078.  It's the sole remaining use of `HighlightListener`, and I don't want to convert something that shouldn't exist in the first place. 

@jessegreenberg Suggestions on how to proceed?

**Comment by pixelzoom** [comment]
I see this comment at the top of TwoSceneSelectionNode.js:

```
 * This was created in support of accessibility.  It may be moved into common code at some point.  Alternatively, it
 * may be obsoleted and replaced with a regular, accessible radio button group.  See
 * https://github.com/phetsims/balloons-and-static-electricity/issues/213 for the discussion on this topic.
```

Is RadioButtonGroup not accessible?

**Comment by pixelzoom** [comment]
I moved scenery-phet HighlightListener to a private class in TwoSceneSelectionNode.js.   So now BASE owns it and can decide how to proceed.  That seemed preferable to trying to replace it in TwoSceneSelectionNode, since TwoSceneSelectionNode shouldn't even exist.

**Comment by jessegreenberg** [comment]
Yes, there is a bit of a history with this. https://github.com/phetsims/balloons-and-static-electricity/issues/213 has some of it, but there were other discussions too. TwoSceneSelectionNode was created as a compromise for that issue. Some did feel that these buttons should be a RadioButtonGroup while others felt that they should not to optimize how they sound with a screen reader.

Understood that this will add to cost when it is time to work PhET-iO in this sim. At that time we will probably bring this up again and hopefully it can be changed to a RadioButtonGroup. Indeed, RadioButtonGroup is accessible. I am going to close this until then, but I will replace ButtonListener with PressListener now.

**Comment by jessegreenberg** [comment]
I removed the HighlightListener, replaced with PressListener and used its isHighlightedProperty.

Closing this issue after https://github.com/phetsims/balloons-and-static-electricity/issues/470#issuecomment-698474577.
